### ISSUE TITLE
Hi DPI not working with correct resolution

### ISSUE THREAD

**Issue by theJavaScripter80** [usr-sse2/RDM#27]
Hi,
I installed RDM and was able to use to scale to some custom resolution, but the problem that the correctresolutions I want to scale to are not working, they do nothing, I'll explain:
My screen is a Samsung C34F791 which is a 34' ultrawide with a resolution of 3440*1440
I can connect my 2018 Macbook 15' (with Radeon VGA Card) to it
As the native resolution of 3440*1440 make text too small for my eyes, I wanted to try one of the following resolutions in hiDpi:
2752 x 1152 hidpi
2560x1080 hidpi
1720 * 720 hidpi

I was able to add them to the menu, but when I click on any of them nothing happens.
When I add them without the hidpi checkbox, I'm able to select them and they work, but ofc that's not what I want, I want them as hiDpi (with the flash sign next to them).

Any ideas what can be wrong?

**Comment by jnooree** [comment]
If you have downloaded the latest release, you should set the resolution to the twice of your interest. For instance, 2752 * 1152 HiDPI will be enabled if you add a resolution of 5504 * 2304 with the HiDPI box checked.

**Comment by theJavaScripter80** [comment]
Thanks I'll try that, what I've tried is 3440*1440 HiDPI, but it didn't work, I clicked it and nothing happened, resolution didn't change, any idea why ?
I was able to get some hidpi resolutions working but they were all 16:9 aspect ratio, not 21:9.

**Comment by jnooree** [comment]
Well, what is your macOS version? If it is above Catalina (10.15), you might need to use the HEAD version. Note that you should install XCode and XCode command line tools.

**Comment by theJavaScripter80** [comment]
I'm running on Mojave (10.14.6), but why would I need xCode? I just downloaded the zip, unzipped into "Applications" and when I click RDM icon in Applications it opens RDM in system bar, and seems to work for some resolutions

**Comment by jnooree** [comment]
Sorry for confusion. I was meant that if you were trying to install the HEAD version, then you would need the Xcode etc. Try to reboot your mac after adding the resolutions.

**Comment by theJavaScripter80** [comment]
Thanks, no probs. I tried and still some resolutions work and some don't. I'm surprised that the ones that work are 16:9 but the 21:9 hiDpi don't work.

**Comment by theJavaScripter80** [comment]
> If you have downloaded the latest release, you should set the resolution to the twice of your interest. For instance, 2752 * 1152 HiDPI will be enabled if you add a resolution of 5504 * 2304 with the HiDPI box checked.

So I added a resolution of 5504 * 2304 with the HiDPI box checked, but it appears in the menu without the flash sign, can you help me with this please?

**Comment by jnooree** [comment]
Yes, if you have added the larger one and it does appear on the menu, then you might find the lower one, 2752 * 1152 with lightning sign on the menu, too. The larger one is just a fake resolution so normally you don't have to use it.
